openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js

```diff
@@ -11,8 +11,6 @@
  *  limitations under the License.
  */
 
-// / <reference types="cypress" />
-
```

Comment:
why we are removing this line? it helps to provide suggestions in Cypress
https://docs.cypress.io/guides/tooling/IDE-integration

Reply:
No worries, I will add it.